src/main/drivers/accgyro/accgyro_mpu.c

```diff
@@ -410,17 +410,15 @@ static bool detectSPISensorsAndUpdateDetectionResult(gyroDev_t *gyro, const gyro
     }
 
     // Detection failed, disable CS pin again
-
-    spiPreinitByTag(config->csnTag);
-
+    ioPreinitByTag(config->csnTag, IOCFG_IPU, PREINIT_PIN_STATE_HIGH);
```

Comment:
(I don't like this code duplication, but it is only small part of preinit problem.) 

Maybe it should be 
```suggestion
    ioPreinitByIo(gyro->dev.busType_u.spi.csnPin, IOCFG_IPU, PREINIT_PIN_STATE_HIGH);
```
for consistency - `IOConfigGPIO` did use that.